client/components/quizSingle/styles.js

```diff
@@ -0,0 +1,178 @@
+import styled from "styled-components";
+import { breakpoint } from "../../frontend-config";
+import { PrimaryButton, riseUp } from "../shared/styles";
+
+export const QuizHeader = styled.section`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin: 0;
+  background: ${props => props.theme.colors.grey};
+`;
+
+export const QuestionHeaderContainer = styled.section`
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  width: 90%;
+  margin: 10px auto 40px auto;
+  transition-duration: 350ms;
+  animation: ${riseUp} 400ms ease-in-out;
+  animation-fill-mode: forwards;
+  animation-delay: ${props => props.animationDelay};
+
+  @media (min-width: ${breakpoint("md")}) {
+    max-width: 62%;
+    margin: 40px auto;
+  }
+`;
+
+export const QuestionProgressBar = styled.div`
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  width: 100%;
+  justify-content: center;
+  height: 16px;
+  margin-bottom: 40px;
+  border: 2px black solid;
+  border-radius: 10px;
+
+  @media (min-width: ${breakpoint("md")}) {
+    width: 95%;
+    max-width: ${breakpoint("maxWidth")};
+    margin-bottom: 40px;
+  }
+`;
+
+export const QuestionProgressBarFiller = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  width: 100%;
+  height: 10px;
+  border-radius: 10px;
+  background: rgb(24, 226, 140);
+  background: linear-gradient(
+    90deg,
+    rgba(24, 226, 140, 1) ${props => props.completion}%,
+    rgba(255, 255, 255, 1) ${props => props.completion}%
+  );
+`;
+
+export const QuestionProgressBarText = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: flex-end;
+  align-items: right;
+  width: 100%;
+  margin-top: 40px;
+  font-weight: bolder;
+
+  @media (min-width: ${breakpoint("md")}) {
+    width: 95%;
+    max-width: ${breakpoint("maxWidth")};
+  }
+`;
+
+export const AnswersTileSection = styled.section`
+  position: relative;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, min(98%, 375px));
+  grid-gap: 20px;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin: 10px auto;
+
+  @media (min-width: ${breakpoint("md")}) {
+    grid-template-columns: repeat(2, min(45%, 406px));
+    grid-gap: 50px 30px;
+    justify-items: center;
+    width: 95%;
+    max-width: ${breakpoint("maxWidth")};
+    margin: 25px auto;
+  }
+`;
+
+export const AnswerTileContainerLink = styled.a`
+  width: 100%;
+`;
+
+export const AnswerTileContainerStyled = styled.div`
+  opacity: 0;
```

Comment:
Is there a reason for this opacity? 🙂

Reply:
I didn't think about it, i tried to reuse some of already styled elements, so it may be unnecessary in this case. I'll test it.

Reply:
It really did nothing, it is now removed